packages/grafana-data/src/field/fieldColor.ts

```diff
@@ -18,6 +19,7 @@ export interface FieldColorMode extends RegistryItem {
   getColors?: (theme: GrafanaTheme2) => string[];
   isContinuous?: boolean;
   isByValue?: boolean;
+  useSeriesNameHash?: boolean;
```

Comment:
thinking on this a bit more and I think we can remove the `Hash` part of name as that is more or less an implementation detail
```suggestion
  useSeriesName?: boolean;
```

Reply:
Done